plugins/BEdita/Core/src/Model/Entity/ObjectEntity.php

```diff
@@ -185,6 +185,33 @@ protected function getRelationships()
                 ],
                 true
             );
+            $entityDest = $this->getTable()->associations()->getByProperty($relationship)->getTarget()->newEntity();
+            if ($entityDest instanceof JsonApiSerializable) {
+                $destObj = $entityDest->jsonApiSerialize(JsonApiSerializable::JSONAPIOPT_BASIC);
+                if (empty($destObj['type'])) {
+                    $options = ['name' => $relationship];
+                    $result = TableRegistry::get('ObjectTypes')
```

Comment:
Is there a way to avoid this query (performed on every item, I guess) usgin object types cache?